app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def post_path(post)
+    "post/#{post.friendly_id}"
+  end
```

Comment:
We still don't need this helper, it's possible to [name the `post` route on `routes.rb`](http://guides.rubyonrails.org/routing.html#naming-routes). Ex:

```diff
- get 'post/:slug', to: 'posts#post'
+ get 'post/:slug', to: 'posts#post', as: :post
```

This will give us both `post_path` and `post_url` helpers